projects/netgrif-components/src/lib/data-fields/i18n-field/i18n-text-field/i18n-text-field.component.scss

```diff
@@ -3,3 +3,121 @@
     margin: 0 auto;
     width: 100%;
 }
+
+.selected-language-value {
+    outline: none;
+    padding: 9px 0;
+    width: calc(100% - 56px);
```

Comment:
min-width ? IDK
maybe use clamp() ??

Reply:
@machacjozef said it's ok to resolve this.